README.md

```diff
@@ -1,2 +1,68 @@
 # monoetna
 mono-repository version of etna projects
+
+## Docker
+
+Use the `Makefile` at the top of the repo to easily start and stop a set of all etna services that are dockerized.
+
+`make help`
+`make up`
+`make down`
+`make logs`
+
+You will want to use per-project `Makefile`s to access proejct specific databases and bash consoles. eg: `make -C janus help`
+
+### Docker environment setup
+
+You'll want to setup your system's `/etc/hosts` file to support mapping subdomain https urls.  The docker
+environment runs an apache server that terminates SSL with self signed certs currently.  You'll want entries for
+each service like the following:
+
+```
+janus.development.local 127.0.0.1
+metis.development.local 127.0.0.1
+magma.development.local 127.0.0.1
+timur.development.local 127.0.0.1
+```
+
+In addition, you'll want to configure some users and projects into your janus environment.
+
+```bash
+# Make sure services are already running with make up in top level directory.
+make -C janus bash
+○ → ./bin/janus add_project 'test-project' 'Test Project'
+○ → ./bin/janus add_user some.guy@ucsf.edu Zach Collins password
+○ → ./bin/janus permit some.guy@ucsf.edu test-project administrator
+```
+
+## Workflow
+
+1. Create a branch in monoetna as per your feature.
+2. Write code that cross cuts all repositories
+3. Push your changes to master, aka the monoetna repo.
+4. Create PR in monoetna repo, showing the entire change set
+5. Merge to master on monoetna
+6. __!! run `./bin/pull-subtrees` to consume changes from child repos.__  If you forget this step, your push may fail.
```

Comment:
The monorepo seems like a great idea. Do people ever automate these two steps somehow so that the child repos are always in sync? Or does that potentially cause more problems?

Reply:
And I should have finished the README first. I see your note about automating in the future!

Reply:
I definitely intend to add automation for the pushing and pulling!  One thing at a time though.